app/main.py

```diff
@@ -2,24 +2,64 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: any, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: any, owner: any) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: any, value: int) -> None:
+        if not self.min_amount <= value <= self.max_amount:
+            raise ValueError(
+                f"The value must be in the range "
+                f"{self.min_amount} to {self.max_amount}"
+            )
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, height: int, weight: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: int, height: int, weight: int) -> None:
+        self.age = age
+        self.height = height
+        self.weight = weight
 
 
 class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+    age = IntegerRange(4, 14)
+    height = IntegerRange(80, 120)
+    weight = IntegerRange(20, 50)
```

Comment:
The ChildrenSlideLimitationValidator and AdultSlideLimitationValidator classes are well implemented, but they don't actually validate anything. Consider adding a 'validate' method that checks if the given age, height, and weight are within the valid ranges.